### DISCUSSION TITLE
Change location\account where backup files are uploaded to OneDrive.

### DISCUSSION THREAD

**Discussion by MisterTFM** [lavinir/hassio-onedrive-backup#50] [General]
After configuring hassio-onedrive-backup, and authenticating to a OneDrive account, can the backups be uploaded to a different MicroSoft OneDrive account?

**Comment by lavinir** [comment]
The only way to re-initate authentication would be to remove and re-install the addon. Then you can authenticate with the account you want.

**Comment by mps222** [comment]
I see following situation.
What is the problem ?

Authorization to OneDrive has been given
![Screenshot 2024-10-14 094446](https://github.com/user-attachments/assets/05cd36da-5683-41d9-b25b-4bf4f43c4882)
![Screenshot 2024-10-14 094446](https://github.com/user-attachments/assets/9a23fdc2-6773-4937-9d66-cd56e0bcde5a)

**Comment by lavinir** [comment]
It's stuck like this? Hard to say what's going on without seeing any logs. Please go ahead and open an issue if relevant.
